### DISCUSSION TITLE
Video trimming

### DISCUSSION THREAD

**Discussion by bergagi1** [bonsai-rx/bonsai#1310] [Q&A]
Hello,

I have long videos containing 5 test trials. I want to create 5 avi. files start/ending at exact frames for analysis (in DeepLabCut). 
I have tried the option attached, but the resulting files move following the the speed I use to scroll though the video.

![Screenshot 2023-03-29 150131](https://user-images.githubusercontent.com/87769998/228543631-5fafcc5b-5f05-47e7-9ab2-4a3afb9058a3.png)
[SelectFrames.bonsai.zip](https://github.com/bonsai-rx/bonsai/files/11101130/SelectFrames.bonsai.zip)

Any suggestion on how to improve this?

Thank you

**Comment by bruno-f-cruz** [comment]
Check this post from the previous google groups:
https://groups.google.com/g/bonsai-users/c/uiUcQdKxpj4/m/93UWoOUCBQAJ

**Comment by bergagi1** [comment]
Thanks Bruno!
